README.md

```diff
@@ -83,6 +83,7 @@ dexcount {
     printAsTree = false
     orderByMethodCount = false
     verbose = false
+    exportAsCSV = false
```

Comment:
About that I don't know to be honest, writing those CSV files does not hurt and maybe you also want them locally. In any case most builds have the isCI variable `isCI = 'true'.equals(System.getProperty('CI'))` for predexing libraries to make it build faster locally

With that variable we can easily turn it on or off

`exportASCSV = isCI`